src/Makefile.am

```diff
@@ -57,6 +57,7 @@ libt8_installed_headers_forest = \
   src/t8_forest/t8_forest_iterate.h src/t8_forest/t8_forest_partition.h
 libt8_installed_headers_geometry = \
   src/t8_geometry/t8_geometry.h \
+  src/t8_geometry/t8_geometry_handler.hxx \
```

Comment:
Wen can remove the t8_geometry_handler from the internal headers then, right?

Reply:
But it is already removed?